rep-0105.rst

```diff
@@ -81,9 +83,31 @@ The coordinate frame called ``base_link`` is rigidly attached to the
 mobile robot base. The ``base_link`` can be attached to the base in
 any arbitrary position or orientation; for every hardware platform
 there will be a different place on the base that provides an obvious
-point of reference. Note that REP 103 [1] specifies a preferred
+point of reference. Note that REP 103 [1]_ specifies a preferred
 orientation for frames.
 
+earth
+'''''
+
+The coordinate frame called ``earth`` is the origin of ECEF.
+
+This frame is designed to allow the interaction of multiple robots in different map frames.
+If the application only needs one map the ``earth`` coordinate frame is not expected to be present.
+In the case of running with multiple maps simultaneously the ``map`` and ``odom`` and ``base_link`` frames will need to be customized for each robot.
+If running multiple robots and bridging data between them, the transform frame_ids can remain standard on each robot if the other robots_frame_ids are rewritten.
+
+If the ``map`` frame is globally referenced the publisher from ``earth`` to ``map`` can be a static transform publisher.
+Otherwise the ``earth`` to ``map`` transform will usually need to be computed by taking the estimate of the current global position and subtracting the current estimated pose in the map to get the estimated pose of the origin of the map.
+
+In case the ``map`` frame's absolute positon is unknown at the time of startup, it can remain detached until such time that the global position estimation can be adaquately evaluated.
+This will operate in the same way that a robot can operate in the ``odom`` frame before localization in the ``map`` frame is innitialized.
+
+.. figure:: rep-0105/ECEF_ENU_Longitude_Latitude_relationships.svg
+   :width: 640px
+   :alt: Earth Centered Earth Fixed diagram
+   
+   A visualization of Earth Centered Earth Fixed with a tangential ``map`` frame.
+
```

Comment:
With a connected tree, tf should be able to transform a point from any map into any other map. And there's the globally oriented `earth` coordinate frame. 

For long distance planning I would expect it to use the global frame or something else which is specific to the non-euclidian planning. 

When a vehicle is travelling it will need to reparent to a new map periodically, such as when you transition between UTM zones. 


As for planning for long distances. (aka we're going to non-euclidian geometries) the planner will need to understand the manifold it's working in and it can either use the ecef `earth` units or convert to it's own. 

Does that clarify what you're asking? 

(Aside: Would a node that publishes all UTM 60 zones as static transforms from `earth` be useful?)